### PR TITLE
misusing a and an

### PR DESCRIPTION
**Pull request by tim-zju**
Signed-off-by: tim-zju <21651152@zju.edu.cn>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/api/v1.19.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ redirect_from:
  - The API tends to be REST. However, for some complex commands, like `attach`
    or `pull`, the HTTP connection is hijacked to transport `stdout`,
    `stdin` and `stderr`.
- - When the client API version is newer than the daemon's, these calls return an HTTP
```
Comment (AkihiroSuda) [inline docs/api/v1.19.md]: "an aitch tee tee pee" is correct

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: docs/api/v1.19.md (hunk 1 above)
Comment (allencloud) [inline docs/api/v1.19.md]: I am 👎  on this part as well.

File: docs/extend/plugin_api.md [hunk 2]
```diff
@@ -192,5 +192,5 @@ containers which depend on them.
 
 ## Plugins helpers
 
-To ease plugins development, we're providing an `sdk` for each kind of plugins
```
Comment (AkihiroSuda) [inline docs/extend/plugin_api.md]: "an ess dee kay" is correct

**Review by AkihiroSuda — CHANGES_REQUESTED** [review]

File: docs/reference/builder.md [hunk 3]
```diff
@@ -297,9 +297,9 @@ Results in:
      ---> Running in a2c157f842f5
      Volume in drive C has no label.
      Volume Serial Number is 7E6D-E0F7
-    
+
```
Comment (thaJeztah) [inline docs/reference/builder.md]: Be careful with your editor settings; it looks like it's stripping whitespace, which can quickly lead to too many unrelated changes and (in some case) trailing whitespace is added on purpose.

In this case I think it's ok to remove, but try to configure your editor to not automatically strip it (or make sure to revert those changes before committing)

**Comment by stevvooe** [comment]
I don't see any corrections that are correct. Closing.

Thanks for the PR!

**Pull request closed without merging**